Backpack/gradle-maven-push.gradle

```diff
@@ -32,7 +32,7 @@ afterEvaluate { project ->
                 version version
```

Comment:
What do you think we add `-internal` to the version name as well to make it impossible to use it by mistake. e.g version would be `20.0.1-internal`